apps/backend/src/services/polling_service.py

```
@@ -262,66 +336,85 @@ async def _collect_drive_health(self, device: Device) -> None:
-                            capacity_bytes=0,  # Would need to parse size
-                            temperature_celsius=temperature,
-                            health_status=health_status,
-                            smart_status="PASSED" if health_status == "healthy" else "UNKNOWN",
-                            power_on_hours=0,  # Would need SMART parsing
-                            power_cycle_count=0,
-                            reallocated_sectors=0,
-                            pending_sectors=0,
-                            uncorrectable_sectors=0,
+            for drive_data in drive_list:
+                drive_name = drive_data.get("name", "")
+                drive_size = drive_data.get("size", "")
```

Ruff (0.12.2)

Local variable `drive_size` is assigned to but never used

Remove assignment to unused variable `drive_size`

(F841)